ingestion/src/metadata/ingestion/ometa/client.py

```diff
@@ -228,7 +228,13 @@ def _one_request(self, method: str, url: URL, opts: dict, retry: int):
                 f"Unexpected error calling [{url}] with method [{method}]: {exc}"
             )
         if resp.text != "":
-            return resp.json()
+            try:
+                return resp.json()
+            except Exception as exc:
+                logger.debug(traceback.format_exc())
+                logger.warning(
+                    f"Unexpected error while returing response in json format - {exc}"
```

Comment:
Please add here as a `logger.debug` the `resp` itself to understand why the JSON parsing failed

Reply:
oh yes 👍